internal/configs/version1/nginx.tmpl

```diff
@@ -49,6 +51,7 @@ http {
         '' $sent_http_grpc_status;
     }
 
+
```

Comment:
nit, extra line (unless it's necessary, in which case we might be better off removing the next `-` :)